ref-exchange/src/account_deposit.rs

```diff
@@ -70,8 +70,13 @@ impl AccountDeposit {
 
     /// Registers given token and set balance to 0.
     /// Fails if not enough amount to cover new storage usage.
-    pub(crate) fn register(&mut self, token_id: &AccountId) {
-        self.tokens.insert(token_id.clone(), 0);
+    pub(crate) fn register(&mut self, token_ids: &Vec<ValidAccountId>) {
+        for token_id in token_ids {
```

Comment:
I moved the loop here - it's an optimization -> with that, we call the `assert_storage_usage()` only once.